index.html

```diff
@@ -269,27 +241,30 @@
     </thead>
     <tbody>
       <tr><td>miss</td><td></td><td></td></tr>
-      <tr><td>dc</td><td></td><td>atl</td></tr>
       <tr><td>db</td><td>53</td><td></td></tr>
-      <tr><td>app</td><td>47.2</td><td>customView</td></tr>
+      <tr><td>app</td><td>47.2</td><td></td></tr>
+      <tr><td>customView</td><td></td><td></td></tr>
+      <tr><td>dc</td><td></td><td>atl</td></tr>
       <tr><td>total</td><td>123.4</td><td></td></tr>
     </tbody>
   </table>
 
-  The above header fields communicates five distinct metrics that illustrate all the possible ways for the server to communicate data to the user agent: metric name only, metric with value, metric with value and description, and metric with description. For example, the above metrics may indicate that for `example.com/resource` fetch:
+  <p>The above header fields communicates five distinct metrics that illustrate all the possible ways for the server to communicate data to the user agent: metric name only, metric with value, metric with value and description, and metric with description. For example, the above metrics may indicate that for `example.com/resource.jpg` fetch:</p>
 
-    * There was a cache miss.
-    * The request was routed through the "atl" datacenter ("dc").
-    * The database ("db") time was 53 ms.
-    * The application server ("app") took 47.2ms to process "customView" template or function.
-    * The total time for the request-response cycle on the server was 123.4ms, which is recorded at the end of the response and delivered via a trailer field.
+  <ol>
+    <li>There was a cache miss.</li>
+    <li>The request was routed through the "atl" datacenter ("dc").</li>
+    <li>The database ("db") time was 53 ms.</li>
+    <li>The application server ("app") took 47.2ms to process "customView" template or function.</li>
+    <li>The total time for the request-response cycle on the server was 123.4ms, which is recorded at the end of the response and delivered via a trailer field.</li>
+  </ol>  
 
-  The application can collect, process, and act on the provided metrics via the provided JavaScript interface:
+  <p>The application can collect, process, and act on the provided metrics via the provided JavaScript interface:</p>
 
   <pre class="example js">
   window.performance
     .getEntriesByName('https://example.com/resource.jpg')
-    .filter(({ name }) => name === "server")
+    .filter(({ entryType }) => entryType === 'server')
```

Comment:
That's no longer true, is it? Users will need to filter on "resource" and then iterate over their ST entries, no?